src/modules/isValidCarNameString.js

```diff
@@ -0,0 +1,35 @@
+import { NUMBER, CHAR } from '../constant/constants';
+
+const hasNotBlank = (stringArr) => {
+  const noneBlank = stringArr.every((elem) => !elem.includes(CHAR.BLANK));
+
+  return noneBlank;
+};
+
+const isValidLength = (stringArr) => {
+  const isInOfRange = stringArr.every((elem) => {
+    const greaterThanZero = elem.length > NUMBER.ZERO;
+    const maxLengthOrLess = elem.length <= NUMBER.MAX_NAME_LENGTH;
+    const isValid = greaterThanZero && maxLengthOrLess;
+
+    return isValid;
+  });
+
+  return isInOfRange;
```

Comment:
이 메서드도 `isInOfRange`라는 변수 선언 대신 바로 반환하는 것은 어떻게 생각하시나요?


```suggestion
  return stringArr.every((elem) => {
    const greaterThanZero = elem.length > NUMBER.ZERO;
    const maxLengthOrLess = elem.length <= NUMBER.MAX_NAME_LENGTH;
    const isValid = greaterThanZero && maxLengthOrLess;

    return isValid;
  });
```